package/src/constructs/browser-check.ts

```diff
@@ -35,8 +35,11 @@ export class BrowserCheck extends Check {
       this.script = script
     } else if ('entrypoint' in props.code) {
       const entrypoint = props.code.entrypoint
+      const absoluteEntrypoint = path.isAbsolute(entrypoint)
+        ? entrypoint
+        : path.join(this.__checkFilePath, entrypoint)
```

Comment:
`path.resolve` should achieve the same behavior ([docs](https://nodejs.org/api/path.html#pathrelativefrom-to))

```suggestion
      const absoluteEntrypoint = path.resolve(this.__checkFilePath, entrypoint)
```